### PR TITLE
Get rid of tox

### PR DESCRIPTION
**Pull request by willdurand**
Fixes https://github.com/mozilla/addons-server/issues/16149
Fixes https://github.com/mozilla/addons-server/issues/16445

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -488,55 +497,134 @@ commands:
             - run:
                 name: Wait for memcached
                 command: dockerize -wait tcp://localhost:11211 -timeout 1m
+      - run: pip install --no-deps -r requirements/ci_base.txt
```
Comment (diox) [inline .circleci/config.yml:500]: With the removal of `tox`, we shouldn't need `ci_base.txt` anymore. Running `make install_python_test_dependencies` should install `system.txt` and `tests.txt` which should contain everything we need.

Comment (willdurand) [reply .circleci/config.yml:500]: So, we should move `toml` to `system.txt` I guess?

Comment (willdurand) [reply .circleci/config.yml:500]: Do we still need `wheels` and `setuptools`?

Comment (diox) [reply .circleci/config.yml]: I would move `wheel` to `system.txt`.

Comment (diox) [reply .circleci/config.yml]: and `toml` is already in `tests.txt` so it should be ok.

Comment (willdurand) [reply .circleci/config.yml]: > I would move `wheel` to `system.txt`.

looks like it's not even needed?

Comment (diox) [reply .circleci/config.yml]: Looks like you're right!

**Review by diox — APPROVED** [review]

**Pull request merged**